io/disk/arcconf/arcconf_cntl_oper.py

```diff
@@ -83,7 +83,7 @@ def setUp(self):
                 http_repo = "%s%s.rpm" % (self.http_path, self.tool_name)
                 self.repo = self.fetch_asset(http_repo, expire='10d')
                 cmd = "rpm -ivh %s" % self.repo
-            if process.system(cmd, ignore_status=True, shell=True) == 0:
+            if process.system(cmd, ignore_status=True, shell=True) != 0:
```

Comment:
Yes, agreed!! I was doing few experiments while I got the travis error, and forgot to delete this. Thanks.